rpcs3/rpcs3qt/memory_viewer_panel.cpp

```diff
@@ -1244,3 +1247,23 @@ void memory_viewer_panel::ShowImage(QWidget* parent, u32 addr, color_format form
 		f_image_viewer->setFixedSize(f_image_viewer->sizeHint());
 	});
 }
+
+void memory_viewer_panel::ShowAtPC(u32 pc)
+{
+	const u32 id = idm::last_id();
```

Comment:
Don't use idm::last_id in this context. IDM is used in other places by the main thread in Qt code.

Reply:
> Don't use idm::last_id in this context. IDM is used in other places by the main thread in Qt code.

What's a good way to grab the last opened memory viewer then?

Reply:
You can use a struct saved via g_fxo

Reply:
> You can use a struct saved via g_fxo

Gotcha. Should be good to go now.